clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/PartitionHandlingBuilder.java

```diff
@@ -49,6 +50,7 @@
     public PartitionHandlingConfiguration getValue() {
         return new ConfigurationBuilder().clustering().partitionHandling()
                 .whenSplit(this.enabled ? PartitionHandling.DENY_READ_WRITES : PartitionHandling.ALLOW_READ_WRITES)
+                .mergePolicy(MergePolicy.NONE)
```

Comment:
This is the default.

Reply:
Yes - this is intentional.  The logic here should match:
https://github.com/infinispan/infinispan/blob/9.2.x/core/src/main/java/org/infinispan/configuration/cache/PartitionHandlingConfigurationBuilder.java#L33